src/app/user/user.component.html

```diff
@@ -0,0 +1,44 @@
+<h3>User Information</h3>
+
+<form [formGroup]="form" (ngSubmit)="submit()">
+
+  <mat-form-field>
+    <mat-label for="name">Name</mat-label>
+    <input matInput formControlName="name" type="text" id="name">
+  </mat-form-field>
+
+  <mat-form-field>
+    <mat-label for="age">Age</mat-label>
+    <input matInput formArrayName="age" type="number" id="age" min="1">
```

Comment:
Maybe you can add a max too. Just to avoid 100000

Reply:
Done